AVAL-Spec.yml

```diff
@@ -309,6 +375,39 @@ paths:
           description: "Provided AvaL Document ID is already in use."
         500:
           description: "An unexpected error occurred. Details may be included within the response body."
+  /avalmatchings/{avalId}/avaltransactions/{transactionId}/extendedInformation:
+    post:
+      tags:
+        - "AvaL Transactions"
+      summary: "Update an existing AvaL Transaction with new Extended Information."
+      description: ""
+      operationId: "postAvalTransactionExtendedInformation"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: "avalId"
+          in: "path"
+          description: "ID of AvaL Transaction to which the Extended Information should be assigned."
+          required: true
+          type: "string"
+          format: "uuid"
+        - in: "body"
+          name: "body"
+          description: "Extended Information in the form of an AdditionalContent object."
+          required: true
+          schema:
+            $ref: "#/definitions/AdditionalContent"
```

Comment:
Basically, the content of the _AdditionalContent_ objects can be freely defined. As long as it is coordinated with the receiving partner. Therefor any business logic can be performed after receiving any AdditionalContent. But changing core attributes appears to be a sensitive topic, because in this way the mandatory attributes (depending on the transaction status) can be manipulated.

It is planned to define a set of standardized AdditionalContent types. Part of this definition could then be that there is a notation with which you can change core attributes. For example, a prefix like **_CF_ <attribute name>_** would be imaginable for your use case. But of course changes to attributes like the AvaL ID or the transaction state should be forbidden.

Reply:
Yes, i agree with you. We should define that somewhere. I like the CF-Prefix idea.
This recommendation should be part of this Issue. As mentioned in the AVAL-AG-Entwicklung-Protokoll 17.11.22 the main issue is specific for updaing Core-Fields (orderNumberClient):
"Es ist im AvaL Standard nicht vorgesehen, dass der Auftraggeber bei einer
Selbstbeauftragung durch den Auftragnehmer mit einer eigenen Bestellnummer
antwortet. - der Auftragnehmer hat keine Bestellnummer des Auftraggebers.“

Reply:
I documented the use of the CF prefix in the issue #16. If the technical solution solves your problem and there are no further comments, then please give your approval and I'll merge the pull request.